util/pace/util/grid/generation.py

```diff
@@ -274,13 +275,13 @@ def __init__(
         # for the selected floating point precision
         self._agrid = None
         self._np = self._grid_64.np
-        self._dx = None
-        self._dy = None
+        self._dx: Optional[util.Quantity] = None
+        self._dy: Optional[util.Quantity] = None
         self._dx_agrid = None
         self._dy_agrid = None
         self._dx_center = None
         self._dy_center = None
-        self._area = None
+        self._area: Optional[util.Quantity] = None
```

Comment:
Should these be optionals now that we aren't assigning them in the from_external method?

Reply:
These can be reverted to just = None